src/sys/personality.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Result;
+use crate::errno::Errno;
+
+use libc::{self, c_ulong};
+
+libc_bitflags! {
+    pub struct Persona: c_ulong {
+        ADDR_COMPAT_LAYOUT as c_ulong;
```

Comment:
N.B.: These casts won't be necessary if/when https://github.com/rust-lang/libc/pull/1974 lands.